.github/workflows/ci.yml

```diff
@@ -40,15 +40,25 @@ jobs:
     strategy:
       matrix:
         # Test of these containers
-        container: ["ubuntu-dev:20", "alpine-dev:latest"]
+        container: ["ubuntu-dev:20", "alpine-dev:latest", "ubuntu-dev:24"]
         build-type: [Debug, Release]
         compiler: [{ cxx: g++, c: gcc }]
         cxx_flags: ["-Werror"]
+        sanitizers: ["Plain"]
```

Comment:
I would rename Plain to NoSanitizers.
Its easier to understand when looking on the actions this way

Reply:
Actually I would use "ON" / "OFF" this way you can use matrix.sanitizers

Reply:
I don't want to use `ON/OFF` because it has no meaning when you read it https://github.com/dragonflydb/dragonfly/actions/runs/12788815584/job/35650888951 

I used "Plain" for "Plain Build" but "NoSanitizers" is better so I will use that